ui/src/components/connections/ConnectionsTable.tsx

```diff
@@ -14,6 +14,7 @@ import {
 import { useCallback, useEffect, useState } from "react";
 import { generatePath, useNavigate, useSearchParams } from "react-router-dom";
 
+import { positiveIntegerFromStringParser } from "../../adapters/parsers";
```

Comment:
We always use the absolute path on imports.

Reply:
Good catch. Changed